docs/library/network.PPP.rst

```diff
@@ -1,6 +1,11 @@
 .. currentmodule:: network
 .. _network.PPP:
 
+.. note:: Currently only the esp32 port has PPP support enabled in the default
+          build. PPP support can be enabled in custom builds of the stm32 and
+          rp2 ports by enabling networking support and setting
+          ``MICROPY_PY_NETWORK_PPP_LWIP`` to 1.
+
```

Comment:
IMO this should be moved down after the main heading, after the first paragraph.  Note that the first paragraph below already states "only available on selected ports".

Reply:
Have moved it below the first paragraph. I also removed the "selected ports and boards" sentence (suggest we add something like this back if we enable on some other ports & boards in the future.)